src/backend/api.js

```diff
@@ -8,15 +8,8 @@ export const login = async (user) => {
   } else {
     call = axios.get('/auth/login')
   }
-  try {
-    const { data } = await call
-    return data
-  } catch (err) {
-    if (err.response.code === '401') {
```

Comment:
It's not safe to remove the 401 check which was deliberately introduced here https://github.com/wbstack/ui/commit/00120ef2243e70aab8f9a30eab02a03d6899a689

When loading the application, a call against login is made to check whether the user is already logged in. If a 401 is returned, this means the user is not, but it's an expected outcome, so `null` is being returned instead of throwing an error.